src/desktop/main.cc

```diff
@@ -370,8 +368,9 @@ MAIN {
         return;
       }
 
-      if (message.name == "getConfig") {
-        window->resolvePromise(seq, OK_STATE, _settings);
+      if (message.name == "config") {
```

Comment:
for a friendlier API we could support JSON if `key` is not given:

```c++
if (message.name == "config") {
  if (message.has("key")) {
    auto key = message.get("key");
    window->resolvePromise(seq, OK_STATE, app.appData[key]);
  } else {
    window->resolvePromise(seq, OK_STATE, JSON::Object(app.appData).str());
  }
  return;
}
```

Reply:
I think in needs to be wrapped in `encodeURIComponent` as well